### ISSUE TITLE
events handlers not hooked up?

### ISSUE THREAD

**Issue by tx0c** [flareact/flareact#121]
### Describe the problem:

have two routes, the `pages/index.js` and `pages/about.js` with button's click event handler to change some internal state,

if user comes to the front page, and navigate to the about page, then the button's click event handler is working well;

if user directly goes to `/about` route, then **the button's click event handler is not responding**

### Link to a minimal reproduction:

the `pages/index.js`

```js
import Link from "flareact/link";

export default function Index() {
  return (<>
      <Link href="/about">
        <a className="text-blue-500 hover:text-blue-300 hover:underline">About</a>
      </Link>

    <h1>
      You're running React on the Edge!
    </h1>
  </>);
}
```

and `pages/about.js`

```js
import { useState } from 'react';

export default function About() {
  const [ clicks, setClicks ] = useState(0);

  return (<>
    <h1>About</h1>
    <button className="px-2 text-blue-500 bg-purple-100 hover:text-blue-300 border border-red-100 rounded-md shadow" onClick={() => setClicks(clicks+1)}>clicked {clicks} times.</button>
  </>);
}
```

Running this Demo events at https://events-demo.peregrine.workers.dev/about

1. if you directly visit https://events-demo.peregrine.workers.dev/about then the click button does not work;
2. if visit the front page first: https://events-demo.peregrine.workers.dev/ and click the navigation to `/about` that works

**Comment by tx0c** [comment]
looks similar to #115 hydration does not happen; tried add import "flareact/router" then *fixed*;

**Comment by tx0c** [comment]
Any updates?   it seems very little/no contributions since 2021 :: https://github.com/flareact/flareact/graphs/contributors

![image](https://user-images.githubusercontent.com/49659410/114068349-72321080-9863-11eb-8b09-c54326636450.png)

**Comment by traffisco** [comment]
> Any updates? it seems very little/no contributions since 2021 :: https://github.com/flareact/flareact/graphs/contributors
> 
> ![image](https://user-images.githubusercontent.com/49659410/114068349-72321080-9863-11eb-8b09-c54326636450.png)

Its a bug no one can really reproduce. Clear the contents of the out folder and delete the KV associated to the worker. It will work.

And yes, the maintainer went MIA

**Comment by tj-kev** [comment]
Should be solved in PR #164
